### ISSUE TITLE
Hi can you help me with this?

### ISSUE THREAD

**Issue by RyukAD** [Wykerd/waifu2x-node#4]
Hi,
I have been trying to install this for so long but i am still  unable to after trying for so many hours. The error im facing is this:
/home/agam/.cache/node-gyp/10.24.1/include/node/v8.h:4013:43: note: declared here
 _UNUSED_RESULT MaybeLocal<Value> Call(Local<Context> context,
                                  ^~~~
  CXX(target) Release/obj.target/w2xcjs/src/conv.o
  SOLINK_MODULE(target) Release/obj.target/w2xcjs.node
/usr/bin/ld: cannot find -lw2xc
collect2: error: ld returned 1 exit status
w2xcjs.target.mk:139: recipe for target 'Release/obj.target/w2xcjs.node' failed
make: *** [Release/obj.target/w2xcjs.node] Error 1
make: Leaving directory '/home/agam/flash_images/node_modules/waifu2x-node/build'

**Comment by Wykerd** [comment]
Do you have [waifu2x-converter-cpp](https://github.com/DeadSix27/waifu2x-converter-cpp) installed on your system?

**Comment by RyukAD** [comment]
Hi, 
Yes i did have it installed, but the i missed the linker's directory thingy you specified at the beginning.
I used to install any package / library from my projects directory and they all used to work just fine. However in this particular case I had to first find all the search directories that my linker has. Then i navigated to that particular directory (./usr/bin) and then installed the converter-cpp there.. and it worked! after 5 hours 
Appreciate your reply!
